apps/docs/src/app/core/component-docs/table/table-docs.component.ts

```diff
@@ -117,6 +119,18 @@ export class TableDocsComponent {
             component: 'TableColumnSortExampleComponent',
             code: tableColumnSortTsSrc,
             fileName: 'table-column-sort-example'
```

Comment:
this should be table-column-sorting-example

Reply:
Just seeing this, I'll fix it on the main table branch